### ISSUE TITLE
Test for GPU taint intermittently fails

### ISSUE THREAD

**Issue by aaron-lane** [terraform-google-modules/terraform-google-kubernetes-engine#173]
We added a test for this taint in #101: 

```ruby
{
  effect: "NoSchedule",	
  key: "nvidia.com/gpu",	
  value: "present",	
}
```

Unfortunately, the test intermittently fails, so it was removed in #171. We need to investigate why it fails.

**Comment by chrislovecnm** [comment]
@aaron-lane this should be fixed by https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/200

**Comment by aaron-lane** [comment]
@chrislovecnm right you are! Fixed by #200.
